artsy/eigen
# fix: inquiry notifs

This PR resolves [PHIRE-625] 

### Description

fixes the inquiry notif issue by adding wrapping the pushView with a setTimeout.

How to test:
- make an inquiry via the app
- go to CMS with an account with `sales_admin` rights and reply
- this will trigger a notif to the app
- with the app killed open the notif

Expected result:
- notif should take you to the conversation page



### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀


<details><summary>Changelog updates</summary>

### Changelog updates









#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- fix: inquiry notifs - gkartalis



</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md


[PHIRE-625]: https://artsyproduct.atlassian.net/browse/PHIRE-625?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
